argusclient/client.py

```diff
@@ -386,6 +386,24 @@ def get_user_dashboards(self, ownerName, shared=True, limit=None, version=None):
         assert ownerName, "Expected a owner name"
         return self.argus._request("get", "dashboards", params=dict(owner=ownerName, shared=shared, limit=limit, version=version))
 
+class PermissionsServiceClient(object):
+    """
+    Service class that interfaces with the Argus permissions endpoint.
+
+    There is no need to instantiate this directly, as it is available as :attr:`argusclient.client.ArgusServiceClient.permissions` attribute.
+    """
+    def __init__(self, argus):
+        self.argus = argus
+
+    def get_permissions_for_entities(self, entityIds):
+        """
+        Gets permissions that are associated with the given entity id's.
+
+        :return: a dict of entity id's mapped to a list of :class:`argusclient.model.Permission` objects
+        """
+        return self.argus._request("post", "permission/entityIds", dataObj=entityIds)
```

Comment:
I can't find any documentation on this endpoint, but there seems to be one for retrieving permission for a given ID: https://git.soma.salesforce.com/ArgusMonitoring/Argus/wiki/Permissions-Resource

What I am wondering is, if we need to support the `permission/{entityId}` endpoint, will it be a separate client or will it be added to the above? If it is going to be a separate client, then using the name `PermissionsServiceClient` is more appropriate for it, than for the `permission/entityIds` endpoint, so i would rename the above.

What other requests are supported by `permission/entityIds` endpoint? If nothing else, I guess we could make `PermissionsServiceClient` extend `BaseUpdatableModelServiceClient`, whenever we decide to add full support so the above name would be fine.

Reply:
Oh, the endpoint hasn't actually been released yet. I made the endpoint and the PR has been merged but not released yet. Thought it'd be out by now but there have been delays. This is the PR: https://git.soma.salesforce.com/ArgusMonitoring/Argus/pull/1195
It is now scheduled to be released next next week. 
Would you approve the PR before that or want to wait until that's released? 

Regarding the 2nd paragraph, I think putting the method for permission/{entityId} in this same class makes sense. That's what I had in mind. 

Re: 3rd paragraph - Ok. We need that now anyways because Ill update this class so that the allinfo data is put in the _coll cache